components/scream/tests/coupled/shoc_cld_p3_rrtmgp/shoc_cld_p3_rrtmgp.cpp

```diff
@@ -22,7 +22,7 @@ TEST_CASE("shoc-stand-alone", "") {
   REQUIRE_NOTHROW ( parse_yaml_file(fname,ad_params) );
 
   // run params:
-  const auto& num_iters = ad_params.get<int>("Number of Iterations",5);
+  const auto& num_iters = ad_params.get<int>("Number of Iterations");
```

Comment:
Have you confirmed that we don't need this fix in all of the other AD tests?  I'm pretty sure I added this to each test, but maybe in an open PR?

Reply:
I just saw it, and removed it. I don't want tests that should use testing profile logic to use the wrong iter number just because of some typos.

Reply:
Other tests that do get `num_iters` from the parameter list do not use the testing profiles feature, so it doesn't matter. However, we _could_ make all tests (at least the ad ones) use the feature.

Reply:
I like the idea of using this tool for as many tests as we can.  Of course it's most important to take care of the most expensive ones first.  I'm working on testing in a separate PR  so I could extend this tool to all the other tests there if we want to get this PR in ASAP.